src/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```

Comment:
why have you changed .gitignore, and you shouldn't have pushed files other than files from main/java/core/basesyntax